CHANGELOG.md

```diff
@@ -40,6 +40,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - Added `random(from: Date, upTo: Date) -> Date` method that return radom date in in the specified range [#336](https://github.com/SwifterSwift/SwifterSwift/pull/336) by [akuzminskyi](https://github.com/akuzminskyi).
   - Added `string(withFormat format: String)` method to get a string from a date with the given format.
   - Added `init?(integerLiteral value: Int)` initializer to create date object from Int literal. [#342](https://github.com/SwifterSwift/SwifterSwift/pull/342) by [n0an](https://github.com/n0an).
+- **URL**
+  - Added `queryParmeters` property to get the query parameters from a URL as a dictionary by [nathanbacon](https://github.com/nathanbacon).
```

Comment:
Please include the PR number in the CHANGELOG. Use the previous entries as a reference